Nuget/Ploeh.AutoFixture.Xunit2.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+    <metadata>
+        <id>AutoFixture.Xunit2</id>
+        <version>2.2</version>
+        <title>AutoFixture with xUnit.net v2 data theories</title>
+        <authors>Mark Seemann</authors>
```

Comment:
Thank you; as the code is entirely taken from the v1 project and I'm not adding anything new, that doesn't seem appropriate, though. It's still your creation. :-)